### ISSUE TITLE
Question: Is this supposed to work with Mongo DB 3.2?

### ISSUE THREAD

**Issue by asif-ir** [rwynn/route81#6]
Change Streams weren't introduced until mongo version 3.6 so is it worth setting this up for syncing a mongo 3.2 DB server?

**Comment by rwynn** [comment]
This project is really only supporting 3.6 and above via change streams.  But you can make a change to this [line](https://github.com/rwynn/route81/blob/20fb5145ced1552f7857101648379952c87522dc/route81.go#L1594) to flip it to false.  This would turn on a legacy mode where change events are read directly from the oplog.  This requires the MongoDB user to have access to the collection `oplog.rs` in database `local` and that the MongoDB cluster is setup as a replica set (can be standalone/single instance).

**Comment by asif-ir** [comment]
Yes, noticed those little things. Not sure how stable it really is. There is another project that I found to be more active/popular - [GTM](https://github.com/rwynn/gtm). Currently selected that. 

The use is that we are migrating a legacy ruby on rails project (which writes to a mongo 3.2 in prod) to newer technologies and during the migration, we want to keep on synchronising the existing mongo writes and raise intermediate events to transform and change the old document to conform to the new schema (Massive changes). And currently GTM seems to be working properly.

But we haven't finalised any tool yet even though GTM is among the favourites. Any suggestions or considerations (about it being able to handle huge load) that you have?
